lib/_kernel.scm

```diff
@@ -4279,6 +4279,12 @@ end-of-code
             scheme-object   ;; device
    "___os_device_stream_open_process"))
 
+(define-prim ##os-device-open-raw
```

Comment:
See comment below.  This functionality should not be exposed to the Scheme level.